src/utils/url.ts

```diff
@@ -41,6 +41,7 @@ const pathsFrontendSupportsWithoutUuids = [
   /^\/subscriptions\/manage$/,
   /^\/license\/detail\/\d+$/,
   /^\/entity\/repository\/history\/\d+$/,
+  /^\/event\/history(\/\d+)?$/,
```

Comment:
Ah so it really belongs there. I decided against it, because it uses Uuids and it's called `pathsFrontendSupportsWithoutUuids`…